src/views/InteractionDesigner.vue

```diff
@@ -296,13 +299,21 @@ export class InteractionDesigner extends mixins(Lang, Routes) {
           this.setIsBlockEditorOpen(true)
         }
       }, 500)
+    }
+    console.debug('Vuej tree interaction designer mounted!')
+  }
 
-      // get the interaction-designer-content positions, will be used to set other elements' position in the canvas (eg: for block editor)
-      if (this.activeFlow && this.$refs['interaction-designer-contents'] !== undefined) {
-        this.setInteractionDesignerBoundingClientRect((this.$refs['interaction-designer-contents'] as Element).getBoundingClientRect())
+  beforeDestroy() {
+    window.removeEventListener('scroll', this.onScroll, true)
+  }
+
+  onScroll(e) {
```

Comment:
added a scroll listener to help us updating the config `top` later. This is needed in case the voto5 page title gets hidden for example